dev/fattest.simplicity/src/componenttest/topology/impl/LibertyClient.java

```diff
@@ -619,6 +619,12 @@ public ProgramOutput startClientWithArgs(boolean preClean, boolean cleanStart,
             if (clientNeedsToRunWithJava2Security()) {
                 addJava2SecurityPropertiesToBootstrapFile(f);
                 Log.info(c, "startClientWithArgs", "Java 2 Security enabled for client " + getClientName() + " because GLOBAL_JAVA2SECURITY=true");
+
+                // If we are running on Java 18+, then we need to explicitly enable the security manager
+                JavaInfo info = JavaInfo.forCurrentVM();
+                if (info != null && info.majorVersion() >= 18) {
```

Comment:
javaInfo is already available in this method so no need to get JavaInfo.forCurrentVM() again.

Reply:
And javaInfo is not null earlier in the class initializer so it cannot be null here so no null check needed.

Reply:
Noted and updated.